### PR TITLE
[Converter/Flatbuf,Flexbuf,Protobuf] Remove duplicated code

### PR DESCRIPTION
**Pull request by myungjoo**
Remove the duplicated function definition.
Create tensor-converter extra subplugin's common util.

Signed-off-by: MyungJoo Ham <myungjoo.ham@samsung.com>

### CODE REVIEW

File: ext/nnstreamer/tensor_converter/tensor_converter_util.h [hunk 1]
```diff
@@ -0,0 +1,23 @@
+/* SPDX-License-Identifier: LGPL-2.1-only */
+/**
+* @file        tensor_converter_util.h
+* @date        26 May 2021
+* @brief       Utility functions for NNStreamer tensor-converter subplugins.
+* @see         https://github.com/nnstreamer/nnstreamer
+* @author      MyungJoo Ham <myungjoo.hamt@samsung.com>
+* @bug         No known bugs except for NYI items
+*/
+
```
Comment (gichan-jang) [inline ext/nnstreamer/tensor_converter/tensor_converter_util.h:10]: Header guard is missing.

File: ext/nnstreamer/tensor_converter/meson.build [hunk 2]
```diff
@@ -1,7 +1,8 @@
 # flatbuffer
 if flatbuf_support_is_available
   converter_sub_flatbuf_sources = [
-    'tensor_converter_flatbuf.cc'
+    'tensor_converter_flatbuf.cc',
```
Comment (jaeyun-jung) [inline ext/nnstreamer/tensor_converter/meson.build:4]: plz update mk file for android build.
NNSTREAMER_CONVERTER_FLATBUF_SRCS and NNSTREAMER_CONVERTER_FLEXBUF_SRCS
https://github.com/nnstreamer/nnstreamer/blob/c079dbcfd687571ecc8a75e48411b91e8e25f531/jni/nnstreamer.mk#L101

Comment (myungjoo) [reply ext/nnstreamer/tensor_converter/meson.build:4]: Added. Thanks!

File: ext/nnstreamer/tensor_converter/tensor_converter_util.h (hunk 1 above)
Comment (myungjoo) [reply ext/nnstreamer/tensor_converter/tensor_converter_util.h:12]: Added. Thanks!

**Review by gichan-jang — APPROVED** [review]
LGTM

**Pull request merged**